dev/cognitiveservices/data-plane/Language/analyzeconversations-authoring.json

```diff
@@ -214,6 +214,101 @@
         "x-ms-long-running-operation": true
       }
     },
+    "/authoring/analyze-conversations/projects/{projectName}/:authorize-copy": {
+      "post": {
+        "description": "Generates a copy project operation authorization to the current target Azure resource.",
+        "operationId": "ConversationalAnalysisAuthoring_CopyProjectAuthorization",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "common.json#/parameters/ProjectNamePathParameter"
+          },
+          {
+            "in": "query",
+            "name": "allowOverwrite",
+            "description": "Whether to allow an existing project to be overwritten using the resulting copy authorization.",
+            "type": "string",
+            "x-ms-parameter-location": "method"
+          },
+          {
+            "$ref": "common.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Payload to send to source resource to initiate project copying.",
+            "schema": {
+              "$ref": "#/definitions/ConversationalAnalysisAuthoringCopyProjectOptions"
```

Comment:
Why is a result model suffixed as "Options"? Is this a typo?

Reply:
@heaths yes that is because the response model for the `:authorize-copy` API is also the request model for the `:copy` API. Do you suggest renaming both? If yes, to what?

Reply:
Not sure how SDK architects will respond, but it makes sense used here.